core/templates/dev/head/components/summary_tile/CollectionSummaryTileDirective.js

```diff
@@ -70,6 +73,11 @@ oppia.directive('collectionSummaryTile', [
           $scope.getStaticImageUrl = function (url) {
             return UrlInterpolationService.getStaticImageUrl(url);
           };
+
+          $scope.collectionIsActive = false;
```

Comment:
Consider $scope.collectionIsCurrentlyHoveredOver -- that might be more clear. "Active" seems like it needs definition.

Reply:
Done!